root/static/scripts/tests/release-editor/edits.js

```diff
@@ -786,14 +786,14 @@ test("mediumCreate edits are not given conflicting positions", function (t) {
         "position": 4,
         "name": "foo",
         "release": "f4c552ab-515e-42df-a9ee-a370867d29d1",
-        "hash": "e886dc4907c701cf89e5e7b5fdebcb521fa04e44"
+        "hash": "aca331e8e3448781852995b146feae853acbaa0e"
```

Comment:
These changed because empty artist credits now get serialized like `[]` instead of `[{name: '', artist: null}]`, or something like that. Not sure why it used to do that but shouldn't really matter.